### PR TITLE
Update test plan for QUARKUS-874

### PR DESCRIPTION
**Pull request by michalvavrik**
### Links

JIRA: https://issues.redhat.com/browse/QUARKUS-874

Quarkus documentation: N/A

### Reminder for considerable topics

 - [x] Make sure you have considered the following areas when preparing the test plan:

   - Logging
   - Tracing
   - Metrics
   - Security
   - OpenAPI
   - Data sources
   - Frontend
   - Qute

### CODE REVIEW

File: QUARKUS-874.md [hunk 1]
```diff
@@ -23,26 +30,24 @@ NOTE: We will prepare steps to get Windows 10 instance to be able to handle situ
...
  - Combinations of Quarkus extensions Test Suite
    - In Github CI - for Pull Requests and Daily runs
- - OpenShift Test Suite
-   - New Jenkins Jobs on selected modules only (the ones using deployment strategies)
-
-### Platform testing
-JVM mode on is the focus for Windows platform. This RFE does not include native executable (Mandrel) or any extension that requires native libraries (like Kafka streams client).
+ - Quarkus QE Test Suite
+   - In Github CI - for Pull Requests and Daily runs
+   - New weekly Jenkins Job running against Quarkus main
```
Comment (rsvoboda) [inline QUARKUS-874.md:35]: Do you have cost estimate for this run?

Comment (michalvavrik) [reply QUARKUS-874.md:35]: This is not a public information. I can count it based on RHBQ TS run, but will need to put it somewhere else.

Comment (michalvavrik) [reply QUARKUS-874.md:35]: @rsvoboda 
misc - 28 min.
spring - 41 min.
database - 2 hr 21 min.
monitoring - 41 min.
messaging - 53 min.
security - 58 min.
http - 1 hr 10 min.
root - 2 hr 41 min.

roughly 9.88 hour per week, please refer to our billing for the cost __per On Demand Windows m5zn.metal Instance Hour_, I think other items are not important to include into counting for their price is too low

Comment (michalvavrik) [reply QUARKUS-874.md:35]: I'll add the time estimate to the plan

File: QUARKUS-874.md [hunk 2]
```diff
@@ -2,17 +2,24 @@
 
 JIRA link: https://issues.redhat.com/browse/QUARKUS-874
 
-The goal is to ensure some level of testing and identify existing limitations on Quarkus side for Windows.
+The goal is to provide production support for JVM mode on Windows, both on bare-metal instances and in OpenShift.
```
Comment (mjurc) [inline QUARKUS-874.md:5]: I think for now the only support for OpenShift we want is verifying that deployment from Windows works, correct? We don't aim specifically to have OCP worker nodes running on Windows machines.

File: QUARKUS-874.md (hunk 1 above)
Comment (mjurc) [reply QUARKUS-874.md:35]: We should add the cost estimates as well as the paper track leading up to us using AWS to infra repo.

File: QUARKUS-874.md (hunk 2 above)
Comment (michalvavrik) [reply QUARKUS-874.md]: Alright, I didn't understand what is requirement for the OpenShift. Updated text, thank you.

File: QUARKUS-874.md [hunk 3]
```diff
@@ -23,26 +30,25 @@ NOTE: We will prepare steps to get Windows 10 instance to be able to handle situ
...
- - Remote Docker machines instances based on RHEL to run the containers with services like DBs / Keycloak / Kafka / Infinispan for Windows jobs
-   - Windows on PSI does not support Linux containers yet, so we want to try to use RHEL instances to run the containers as a workaround
+ - Windows instances will be created from RHEL instances, therefore we need at least one RHEL instance of medium flavor for each job.
+ - Windows instances that requires Docker will be run in AWS, therefore only impact to the current capacity is above-mentioned RHEL instance. 
+ - Windows instances that do not require Docker will be executed next to the RHEL instances, but they are mostly lightweight short-running jobs
+ - Overall test execution for RHEL and Windows jobs will be affected (doubled in the worse case)
  - More jobs pulling artifacts from Artifactory instance
-   - We may need to use more resources for Artifactory
+ - We may need to use more resources for Artifactory
+ - Weekly Test Suite run on AWS will require around 9.88 h per week
```
Comment (rsvoboda) [inline QUARKUS-874.md:50]: This number will grow as the TS size grows, add a note about it.

Comment (michalvavrik) [reply QUARKUS-874.md]: Appreciated!

File: QUARKUS-874.md (hunk 1 above)
Comment (michalvavrik) [reply QUARKUS-874.md:35]: Created MR. Thanks.

**Review by rsvoboda — APPROVED** [review]

**Review by mjurc — APPROVED** [review]

**Comment by mjurc** [comment]
Thanks for the contribution!

**Pull request merged**